pkgs/tools/security/onlykey-agent/default.nix

```diff
@@ -0,0 +1,61 @@
+{ lib
+, python3Packages
```

Comment:
should use python3 instead as that's what exists at the `pkgs` package scope.

Also, overriding python packages currently returns an interpreter